index.html

```diff
@@ -766,14 +770,6 @@
           </p>
         </section>
 
-        <section id="ldpcvpatch">
```

Comment:
The anchor `#ldpcvpatch` is referenced so that needs to be addressed to allow removal. Otherwise I'm :+1: on the change.

Reply:
Updated with: https://github.com/fcrepo/fcrepo-specification/pull/244/commits/cc00a85643e6257d418789c2d9eefdf50361381e